docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -279,8 +279,20 @@ Release
 
       rm -rf ${PWD}
 
-#. Merge the release pull request.
-   Do not delete the release branch because it may be used as a base branch for future releases.
+#. Merge the release branch into master.
+
+   If there are no conflicts, merge the pull request.
+   If there are conflicts; create a new branch, merge forward and create a pull-request of that branch against master.
+
+   .. prompt:: bash $
+
+      git checkout -b merge-release-${VERSION}-FLOC-XXX release/flocker-${VERSION}
+      git pull origin master
+
+   Merging this pull-request will also close the release pull request.
+
+   Unless this is a development release,
```

Comment:
Clarify this, should the release engineer delete the `merge-release-FLOC-XXX` branch? It seems like yes

 Will the next release use the `merge-release` branch as a base? It seems like no